docs/site/Routes.md

```diff
@@ -226,3 +226,57 @@ export class MyApplication extends RestApplication {
   }
 }
 ```
+
+## Mounting an Express Router
+
+If you have an existing [Express](https://expressjs.com/) application that you
+want to use with LoopBack 4, you can mount the Express application on top of a
+LoopBack 4 application. This way you can mix and match both frameworks, while
+using LoopBack as the host. Mounting an Express router on a LoopBack 4
```

Comment:
Lovely 👏 

Can we also add a link to docs explaining how to compose LB4 and Express the other way, where Express is acting as the host and LB4 routes are mounted?